README.md

```diff
@@ -10,10 +18,26 @@ yarn install
 yarn dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+Open <http://localhost:3000> with your browser to see the result.
 
 You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
 
+If this project is run without the orchestration service from the korerorero-reverse-proxy project it will complain of network errors.
+
+## Storybook
+
+To view this project with a mock orchestration service.
+
+This command start storybook and a mock service, so make sure you quit `korerorero-reverse-proxy` before you run it, otherwise there will be a port conflict resulting in network errors.
```

Comment:
```suggestion
This command starts storybook and a mock service, so make sure you quit `korerorero-reverse-proxy` before you run it, otherwise there will be a port conflict resulting in network errors.
```